designer/server/src/main/scala/pl/touk/nussknacker/ui/validation/ParametersValidator.scala

```diff
@@ -30,4 +37,38 @@ class ParametersValidator(modelData: ModelData, scenarioPropertiesNames: Iterabl
       .flatten
   }
 
+  def validate(request: AdhocTestParametersRequest, parameters: Map[String, List[Parameter]])(
```

Comment:
1. Pass `sourceParameters` instead of `request`. `scenarioGraph` is already  translated into `parameters`
2. Rename `parameters` to `parametersDefinition` to avoid confusion with  `sourceParameters

Reply:
Good, done that - thanks :+1: